gce/gce.go

```diff
@@ -382,6 +382,131 @@ func (s *gceOps) DeleteFrom(id, _ string) error {
 	return s.Delete(id)
 }
 
+func (s *gceOps) DeleteInstance(instanceID string) error {
+
+	if s.inst.clusterName == "" || s.inst.clusterLocation == "" {
+		// We delete an instance only if its part of GKE. Not GCE.
+		return fmt.Errorf("running on non-GKE environment. Not deleting instance [%v]", instanceID)
```

Comment:
The delete instance method doesn't need to add a "cluster" restriction and should work on plain GCP itself.

So take in the instance zone as an argument. 

To keep the APIs consistent, do the same for InspectInstance().

Reply:
Hi @harsh-px , we can pass instance zone to DeleteInstance() and InspectInstance() but the problem is the calling API needs to know the zone of that instance then.  Right now there is no way to find out zone of an instance just by its ID. (Instance residing in zone other than where gke SDK is running.)